python/tests/conftest.py

```diff
@@ -0,0 +1,34 @@
+import pytest
+import unittest
+import resource
+import functools
+import os
+
+#from tests import equinor_test
+
+def source_root():
+    path_list = os.path.dirname(os.path.abspath(__file__)).split("/")
+    while len(path_list) > 0:
+        git_path = os.path.join(os.sep, "/".join(path_list), ".git")
+        if os.path.isdir(git_path):
+            return os.path.join(os.sep, *path_list)
+        path_list.pop()
+    raise RuntimeError('Cannot find the source folder')
+
+
+def has_equinor_test_data():
+    return os.path.isdir(os.path.join(source_root(), "test-data", "Equinor"))
+
+
+def pytest_configure(config):
+    if has_equinor_test_data() and not all(x > 2000 for x in resource.getrlimit(resource.RLIMIT_NOFILE)):
+        new_limit = 2048
+        print("Running Equinor tests, and maximum open file handles is: {}, increasing to: {}".format(
+            min(resource.getrlimit(resource.RLIMIT_NOFILE)), new_limit))
+        resource.setrlimit(resource.RLIMIT_NOFILE, (new_limit, new_limit))
+
+
+def pytest_runtest_setup(item):
+    if item.get_closest_marker("equinor_test") and not has_equinor_test_data():
+        pytest.skip("Test requires Equinor data")
```

Comment:
Just a minor thing. Why not replacing this method with something like
```Python
equinor_test = pytest.mark.skipif(lambda: not has_equinor_test_data())
```

This way it is explicit that we are creating a new marker/decorator

Reply:
Considered doing something like that, the reason I chose mark was if we wanted the flexibility to just run the Equinor tests with pytest -m "equinor_test", but if that is less readable I'll be happy to change to your suggestion.